### DISCUSSION TITLE
Would it be possible to use baritone with WNT?

### DISCUSSION THREAD

**Discussion by ghost** [VideoGameSmash12/WNT#8] [Q&A]
and would you be willing to build this into your mod?

**Comment by VideoGameSmash12** [comment]
It can be used with Baritone.

I actually haven't coded it to integrate with Baritone, but nothing prevents the mod from running alongside Baritone if that is installed as a Fabric mod (or included with another mod like Meteor). You are able to run Baritone's commands from the Blackbox's chat window as well.

I don't intend to build Baritone into the mod itself though because that can cause issues if Baritone updates and removes their old versions.
